spikeinterface/sortingcomponents/peak_detection.py

```diff
@@ -271,10 +263,9 @@ class DetectPeakLocallyExclusive:
     engine = 'numba'
     params_doc = DetectPeakByChannel.params_doc + """
     local_radius_um: float
-        For exclusive spatialy.
+        The radius to use for detection across local channels.
```

Comment:
I do not think this is formulated correctly.

Reply:
now better